.github/config.yml

```diff
@@ -80,27 +80,8 @@ checks:
     - publish
     - verify
     required: true
-  windows:
```

Comment:
Should disabling windows-CI be part of the backport?

Reply:
hmm - not sure we are offering that support

in the case of 1.29 i dont think we should either as its not been published anywhere

the problem with retaining support on the other branches is that ci runs in `main` by default so we would need to bring that back there (or hack some different ci path)